cachix/src/Cachix/Client/WatchStore.hs

```diff
@@ -25,7 +26,9 @@ producer store mgr queue = do
 queueStorePathAction :: Store -> PushQueue.Queue -> Event -> IO ()
 queueStorePathAction store queue (Removed lockFile _ _) = do
   sp <- Store.parseStorePath store (encodeUtf8 $ toS $ dropLast 5 lockFile)
-  atomically $ TBQueue.writeTBQueue queue sp
+  filterInvalidStorePath store sp >>= \case
+    Nothing -> return ()
```

Comment:
We should issue a warning here

Reply:
It prints out the usual warning that it skipped over the path.